src/vue-image-compare.vue

```diff
@@ -118,6 +120,11 @@ export default {
             default: false,
             required: false
         },
+        allowRightClick: {
+            type: Boolean,
+            default: true,
```

Comment:
why let right click by default ? you want to be able to move the comparison with right click ?